modules/checkout/src/Annotation/CommerceCheckoutPane.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// @codingStandardsIgnoreFile
```

Comment:
@bojanz Also said that this should be refactored. However, until it is done, I think this should just be kept in.